src/libraries/System.Runtime/tests/System/Reflection/MethodBaseTests.cs

```diff
@@ -56,26 +56,18 @@ public static void TestMethodBody()
             MethodBase mbase = typeof(MethodBaseTests).GetMethod("MyOtherMethod", BindingFlags.Static | BindingFlags.Public);
             MethodBody mb = mbase.GetMethodBody();
             Assert.True(mb.InitLocals);  // local variables are initialized
-#if DEBUG
```

Comment:
I think this can stay as is. As is, the debug build of the test should work on release runtime and vice versa. 

Relaxing the values expected by the test is reducing the coverage.